### ISSUE TITLE
Select all

### ISSUE THREAD

**Issue by jjconti** [humitos/osm-pois#37]
Estaría bueno poder "seleccionar todos" los tipos de puntos con un solo click.

**Comment by humitos** [comment]
¿En qué caso necesitarías esto? Las veces que lo hice, se te llena la pantalla de íconos y complica las cosas.

Y, por otro lado, si estás buscando algo puntual, no querés ver un montón de íconos que no son relevantes.

¿Qué decís?

**Comment by jjconti** [comment]
En lugares donde hay pocas cosas puede ser útil. Caso concreto: busqué mi barrio en osm y dije "a ver que hay cerca". La única opción fue ir marcando de a una las opciones.

**Comment by jjconti** [comment]
O sino podrían crearse grupos de tipos de puntos (los que tienen el mismo color de ícono serían del mismo grupo, por ejemplo) y que con un click te habilite todos los de ese grupo: comida, dinero, hospedaje, son ejemplos de grupos.

**Comment by humitos** [comment]
Sí, me gusta la idea de los diferentes grupos. Creo que tiene más sentido.

Por ejemplo, si necesitás sacar plata: "cajero o banco" te da lo mismo. Así que, estaría bueno poder marcar esos dos juntos.
